### ISSUE TITLE
GTK3 - Module Gtk doesn't seem to have TREE_SORTABLE_UNSORTED_SORT_COLUMN_ID on Ubuntu 14.04

### ISSUE THREAD

**Issue by cmallwitz** [MeanEYE/Sunflower#220]
>>> from gi.repository import Gtk

>>> print (Gtk.TREE_SORTABLE_UNSORTED_SORT_COLUMN_ID)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib/python2.7/dist-packages/gi/module.py", line 320, in __getattr__
    return getattr(self._introspection_module, name)
  File "/usr/lib/python2.7/dist-packages/gi/module.py", line 139, in __getattr__
    self.__name__, name))
AttributeError: 'gi.repository.Gtk' object has no attribute 'TREE_SORTABLE_UNSORTED_SORT_COLUMN_ID'

Gtk.TREE_SORTABLE_DEFAULT_SORT_COLUMN_ID has similar issue.

In file_list.py:1294, if I use -2 instead it seems to do the right thing (or at least not to break in an obvious way)

Interestingly this works as expected:

>>> print (Gtk.DirectionType.TAB_FORWARD)
<enum GTK_DIR_TAB_FORWARD of type GtkDirectionType>

More worringly I can't find any reference to such an issue on the web.

On Ubuntu 15.10 there are no issues.

**Comment by cmallwitz** [comment]
In case this was an issue with my box, verified issue exists on a fresh install of Ubuntu 14.04 too.

**Comment by MeanEYE** [comment]
Yes, there are the following constants available in my Gtk resource file:
```
    TREE_SORTABLE_DEFAULT_SORT_COLUMN_ID = -1
    TREE_SORTABLE_UNSORTED_SORT_COLUMN_ID = -2
```
So they are just numbers. Not entirely sure we need this in the first place. Thanks for reporting. I'll check if this can be replaced by our own constant or just completely ignored.

**Comment by cmallwitz** [comment]
Fixed by PR #222.
